pokemongo_bot/cell_workers/update_title_stats.py

```diff
@@ -69,6 +70,7 @@ def __init__(self, bot, config):
         self.next_update = None
         self.min_interval = self.DEFAULT_MIN_INTERVAL
         self.displayed_stats = self.DEFAULT_DISPLAYED_STATS
+        self.terminal = self.TERMINAL
```

Comment:
This seems a bit repetitive. Why shouldn't you initialise default values here directly without going through these constants?